website/src/components/Header.tsx

```diff
@@ -126,7 +126,9 @@ export default class Header extends React.Component<HeaderProps, any> {
                                     )}
 
                                     <Nav.Link
-                                        className="btn btn-outline-primary ml-3 px-5 py-2"
+                                        className="btn btn-outline-primary ml-3 px-5 py-2 cta-btn"
+                                        data-button-style="2"
```

Comment:
Could we add a constant file as a source of truth to map these numbers to variable names? Magic numbers can be confusing to read without documentation. What do you think about the following?

```js
const buttonStyle = {
  primary: 1,
  outline: 2,
  arrowWithText: 3,
  arrowBoxWithText: 4,
  arrowBoxButton: 5,
}
```

```js
const buttonLocation = {
  nav: 1,
  hero: 2,
  body: 3,
  trySourcegraph: 4,
}
```

```jsx
import { buttonStyle, buttonLocation } from 'tracking'

<button
  data-button-type="cta"
  data-button-style={buttonStyle.primary}
  data-button-location={buttonLocation.hero}
>
    My Button
</button>
```

Reply:
Love it!